README.md

```diff
@@ -0,0 +1,150 @@
+# Sugar AI: Child-Friendly AI Assistant & Python Coding Helper
+
+A dual-purpose AI system combining child-friendly question answering with Python coding assistance using Retrieval-Augmented Generation (RAG).
```

Comment:
What do you mean by "dual-purpose"?

Reply:
It wasn't clear at first what it was, it also doesn't just help with Python, it specifically can help with Sugar and GTK related questions.